### PR TITLE
Branch4

### PR DESCRIPTION
**Pull request by code-labor**

### CODE REVIEW

**Comment by codelabor** [comment]
review test

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ test-repository
 branch1 test
 
 branch2 test
+
+branch4 test
```
Comment (codelabor) [inline README.md]: review test (code)

**Comment by codelabor** [comment]
close and comment

**Comment by codelabor** [comment]
reopen

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ test-repository
 branch1 test
 
 branch2 test
```
Comment (codelabor) [inline README.md]: comment test2 (code)

Comment (code-labor) [reply README.md]: hi!

**Comment by code-labor** [comment]
I'll fix this.

**Pull request closed without merging**